decoder/vaapidecoder_h264.cpp

```diff
@@ -1102,6 +1105,20 @@ bool VaapiDecoderH264::DPB::add(const PicturePtr& picture)
         compPicture->m_picStructure = VAAPI_PICTURE_FRAME;
     }
 
+    if (m_isLowLatencymode) {
+        while(true) {
+            it = find_if(m_pictures.begin(), m_pictures.end(), isOutputNeeded);
```

Comment:
Could we add isLowLatencyOuptutFrame and combine
isOutputNeeded , and isFrame(*it) && (!m_isOutputStarted || ((*it)->m_poc -m_lastOutputPoc == 1) to the function.